frontend/src/metabase/visualizations/components/ChoroplethMap.jsx

```diff
@@ -41,10 +41,10 @@ export function getColorplethColorScale(
 
   const darkColor = Color(color).darken(darken).saturate(saturate);
 
-  const scale = d3.scale
-    .linear()
-    .domain([0, 1])
-    .range([lightColor.string(), darkColor.string()]);
+  const scale = d3.scaleLinear(
+    [0, 1],
+    [lightColor.string(), darkColor.string()],
+  );
```

Comment:
TypeScript is willing to interpret strings as ranges when using the `scaleLinear(domain, range)` api, but it throws a type error when using `scaleLinear().domain(...).range(...)`, as it expects the range to be numbers only.